modules/clientBinaryManager.js

```diff
@@ -3,29 +3,43 @@
 const _ = global._;
 const path = require('path');
 const Q = require('bluebird');
+const fs = require('fs');
+const electron = require('electron');
+const ipc = electron.ipcMain;
+const app = electron.app;
 const got = require('got');
 const Settings = require('./settings');
+const Windows = require("./windows");
 const ClientBinaryManager = require('ethereum-client-binaries').Manager;
 const EventEmitter = require('events').EventEmitter;
 
 const log = require('./utils/logger').create('ClientBinaryManager');
 
 
+const ALLOWED_DOWNLOAD_URLS_REGEX =
+    /^http[s]?:\/\/([^\.]*\.)?ethereum\.org\/(.*)?|^http[s]?:\/\/([^\.]*\.)?bintray\.com\/karalabe\/ethereum\/(.*)?/;
```

Comment:
This pattern doesn't match the current urls as the correct bintray url is bintray.com/*artifact/download*/karalabe.. This one should do the trick:

`
^http[s]?:\/\/([^\.]*\.)?ethereum\.org\/(.*)?|^http[s]?:\/\/([^\.]*\.)?bintray\.com\/artifact\/download\/karalabe\/ethereum\/(.*)?
`

Otherwise it's correct.